projects/robots/boston_dynamics/spot/protos/SpotRightLeg.proto

```diff
@@ -0,0 +1,841 @@
+#VRML_SIM R2020a utf8
+# license: Apache License 2.0.
+# license url: http://www.apache.org/licenses/LICENSE-2.0.
+# documentation url: https://www.cyberbotics.com/doc/guide/spot
+# The "Spot" is a dog robot developed by Boston Dynamics.
+# This PROTO is the definition of right legs.
+# Created by Greg McKechnie under Common Creative license: https://sketchfab.com/3d-models/spot-mini-rigged-5dcbee77730640269cef5bd2587e328a
```

Comment:
```suggestion
# tags: static, hidden
```

Not description required as this is an internal PROTO (not used by the user directly).

Reply:
This is still not fixed!